### PR TITLE
fix: hide prev and next buttons when there is only one image in the p…

### PR DESCRIPTION
**Pull request by dariusmihut**
…rofile image gallery

PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Conditionally hiding the prev, next buttons in the image gallery from the SpaceProfile 

## Git Issues

Closes #

#3242

## Screenshots/Videos

When only one image
image

When multiple images
image

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of our monthly maintainers call (first Monday of the month)

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

**Comment by davehakkens** [comment]
Thanks for the quick fix. Looks good to me!

**Comment by thisislawatts** [comment]
@dariusmihut could we fix this in the ImageGallery component rather than this specific instance of it. It seems preferable for all places where the Gallery displays only one image for us to hide the arrow butons.

**Comment by dariusmihut** [comment]
> @dariusmihut could we fix this in the ImageGallery component rather than this specific instance of it. It seems preferable for all places where the Gallery displays only one image for us to hide the arrow buttons.

very good point, i will

**Comment by dariusmihut** [comment]
@thisislawatts @davehakkens chnged and tested as requested :-)

File: packages/components/src/ImageGallery/ImageGallery.test.tsx [hunk 1]
```diff
@@ -175,4 +176,20 @@ describe('ImageGallery', () => {
...
+
+  it('does not support show next/previous buttons because only one image', async () => {
+    const { queryByRole } = render(
+      <DoNotShowNextPrevButtons
+        {...(DoNotShowNextPrevButtons.args as ImageGalleryProps)}
+      />,
+    )
+
+    // const nextBtn = getByRole('button', { name: 'Next image' })
+    // const previousBtn = getByRole('button', { name: 'Previous image' })
```
Comment (benfurber) [inline packages/components/src/ImageGallery/ImageGallery.test.tsx:188]: I think these comments can be removed?

**Review by thisislawatts — APPROVED** [review]

**Comment by thisislawatts** [comment]
@all-contributors add @dariusmihut for code

**Pull request merged**